source/assets/sass/sass.scss

```diff
@@ -0,0 +1,36 @@
+/*! Sass */
+
+@charset 'utf-8';
```

Comment:
This isn't necessary—Sass chooses whether or not to add a `@charset` to the output independent of whether one exists in the input, and there's no risk of this file being loaded with the wrong encoding. (Plus it has to come before comments in order for it to work.)

Reply:
Fixed in 47bb3b2a1a23252e6c89a508581966b4327b5421